week3/danielday1week3.txt

```diff
@@ -0,0 +1,40 @@
+5.29
+-----
+
+
+
+
+
+6.1 (285/291)
+-----
+a-	y = 123.05 - 8.94(x)
+b-	-8.94, Smoker: 114.11  NonSmoker: 123.05
+c-	Yes, the P=value of 0.0000 is less than 0.05
+
+
+6.2
+-----
+a-	y = 120.07 - 1.93(x)
+b-	-1.93, FirstBorn: 120.07 Others: 118.14
+c-	No, because we get a P-value larger than 0.05
+
+
+
+
+6.3
+-----
+a-	y  = -80.41 + 0.44(gestation) - 3.33(parity) - 0.01(age) + 1.15(height) + 0.05(weight) - 8.40(smoke)
+  
+b-	Gestation: 0.44, Age: -0.01
+
+c-	Once you include other factors, it has an effect on Parity.

```

Comment:
This is really just saying it's different, you didn't give a reason *why* it's different. This doesn't have to hit on domain knowledge, it could just be something like:
"The other variables were masking the effect from parity."